presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -157,6 +163,12 @@ These properties require some explanation:
   queries that can be run, while setting it too high will cause the JVM
   to run out of memory.
 
+* ``query.max-memory=50GB``:
+  The maximum amount of distributed memory that a query may use.
+
+* ``query.max-memory-per-node=1GB``:
+  The maximum amount of memory that a query may use on any one machine.
```

Comment:
should we document that they need more memory than this in the heap, or maybe fail to start on a really bad number